cmd/sonobuoy/app/run.go

```diff
@@ -63,10 +63,6 @@ func (r *runFlags) Config() (*client.RunConfig, error) {
 		GenFile:    r.genFile,
 	}
 
-	if r.genFile != "" && givenAnyGenConfigFlags(&(r.genFlags), allowedGenFlagsWithRunFile) {
```

Comment:
All moved into a prerun method which natively has all the flags available. No longer need to put the `genFlags` flagset inside the `genFlags` object (I probably have the casing wrong, but thats why I'm so happy to get rid of it!